tekton/src/pipelines/taskdefs/fvt-manage/phase3.yml.j2

```diff
@@ -197,3 +197,63 @@
     - fvt-manage-wo-basic
     - fvt-manage-ancestors
     - fvt-manage-automationscripts
+
+# Manage FVT crontask
+- name: fvt-manage-crontask
+  {{ lookup('template', 'taskdefs/fvt-manage/api/taskref.yml.j2') | indent(2) }}
+  params:
+    {{ lookup('template', 'taskdefs/fvt-manage/api/params.yml.j2') | indent(4) }}
+    - name: fvt_test_suite_prefix
+      value: base
+    - name: fvt_test_suite
+      value: crontask
+  runAfter:
```

Comment:
Hi @chiragcsatani Please fix the runAfter for each test. It should maintain the sequence of the tests

Reply:
Exactly.. make sure the runAfter of tests added in phase are the same of other phase 3 tests.

Also, as you added more tests in phase 3, you need to be sure those tests are also the runAfter of all phase 4 tests. Similarly, for tests added in Phase 4, they need to be part of the runAfter of phase 5.

And be sure to update the comments on top of each file, with the test and "{type)" of each test.

Reply:
Thank you Alex. it cleared me how we organise tests. updated